graphlearn_torch/python/partition/base.py

```diff
@@ -807,10 +843,13 @@ def load_partition(
       os.path.join(node_pb_dir, f'{as_str(ntype)}.pt'), map_location=device)
 
   edge_pb_dict = {}
-  edge_pb_dir = os.path.join(root_dir, 'edge_pb')
-  for etype in meta['edge_types']:
-    edge_pb_dict[etype] = torch.load(
-      os.path.join(edge_pb_dir, f'{as_str(etype)}.pt'), map_location=device)
+  if not graph_caching:
```

Comment:
`edge_pd` is necessary even with graph caching, as there exists the case that edges have features (which should be split into multiple partitions). Thus edge_pd should also be loaded here (should be saved when partitioning).